modules/nw-egress-ips-about.adoc

```diff
@@ -26,6 +26,15 @@ An egress IP address is implemented as an additional IP address on the primary n
 [id="nw-egress-ips-platform-support_{context}"]
 == Platform support
 
+// Clarify platform support -
+// https://bugzilla.redhat.com/show_bug.cgi?id=1891515
+
+////
+[IMPORTANT]
+====
+Currently, the egress IP addresses are not compatible with Amazon AWS, Azure Cloud, or any other cloud platform that does not support layer 2 manipulations due to their incompatibility with macvlan traffic.
```

Comment:
I am not sure on the exact wording for this, but we should get rid of `or any other cloud platform that does not support layer 2 manipulations due to their incompatibility with macvlan traffic.`. I am thinking something along the lines of:

```
Currently, the fully integrated egress IP solution of OpenShift is only supported on bare-metal and vSphere. OpenShift's egress IP is not expected to work on or tested on: Amazon AWS, Azure Cloud, and Google GCP which all require additional system administrator setup using the individual cloud console. 
```